docs/06-contributors/workflows.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Development Workflows
+id: workflows
+keywords: [Wing contributors, contributors, workflows]
+---
+
+This topic includes a description of common development workflows for thw Wing project.
+
+## Environment Setup
+
+Here is a list of minimal tools you should install to build the Wing repo in your development
+environment:
+
+* [Node.js] version 18.x or above (we recommend [volta])
```

Comment:
While you're moving this, could you add a note that we require npm v8 (not v9) to build as well?

Reply:
Is there an issue that tracks this issue? (so I can reference, and also understand why we are limited to v8)

Reply:
closest is https://github.com/winglang/wing/issues/1103

It has come up only in slack before otherwise

Reply:
Thanks!